src/languages/es.js

```diff
@@ -432,6 +432,11 @@ export default {
             genericFailureMessage: 'Se produjo un error al invitar al usuario al espacio de trabajo. Vuelva a intentarlo..',
             welcomeNote: ({workspaceName}) => `¡Has sido invitado a la ${workspaceName} Espacio de trabajo! Descargue la aplicación móvil Expensify para comenzar a rastrear sus gastos.`,
         },
+        edit: {
+            // TODO
```

Comment:
TODO: I asked in #espanol [here](https://expensify.slack.com/archives/C21FRDWCV/p1627584361142100)